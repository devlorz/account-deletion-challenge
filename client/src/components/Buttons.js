import PropTypes from 'prop-types'
import React from 'react'

const Buttons = props => {
  const { disabled, onBackButton, onSubmit } = props
  return (
    <div>
      <button onClick={onBackButton}>Back</button>
      <button onClick={onSubmit} disabled={disabled}>
        Next
      </button>
    </div>
  )
}

Buttons.PropTypes = {
  disabled: PropTypes.boolean,
  onBackButton: PropTypes.func,
  onSubmit: PropTypes.func,
}

export default Buttons
