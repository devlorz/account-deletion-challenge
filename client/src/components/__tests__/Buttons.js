import React from 'react'
import { render } from 'react-testing-library'

import Buttons from '../Buttons'

test('snapshot', () => {
  const { container } = render(
    <Buttons disabled={false} onBackButton={jest.fn()} onSubmit={jest.fn()} />
  )
  expect(container.firstChild).toMatchSnapshot()
})
