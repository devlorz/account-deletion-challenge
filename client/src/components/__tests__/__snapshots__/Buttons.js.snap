// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  data-reactroot=""
>
  <button>
    Back
  </button>
  <button>
    Next
  </button>
</div>
`;
